package.json

```diff
@@ -15,7 +15,7 @@
     "build-sass": "rm -rf dist/public/style.css && sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed",
     "copy-locales": "mkdir -p dist && copyfiles -u 1 src/**/*.njk dist/ src/locales/**/** dist/ ",
     "copy-images": "copyfiles -u 3 src/assets/images/** dist/public/images",
-    "minfiy-build-front-end-gov-js": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js src/assets/javascript/application.js -o dist/public/javascripts/application.js -c -m ",
+    "minfiy-build-front-end-gov-js": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js src/assets/javascript/**/**/*.js src/assets/javascript/**/*.js -o dist/public/javascripts/application.js -c -m ",
```

Comment:
This is a bit clunky - for some reason the `**` glob is only expanding one directory level, so this won't work if any deeper directories are added. Suggestions welcome!

Reply:
Should we also correct the spelling mistake in 'minfiy' as well?